docs/DeveloperGuide.md

```diff
@@ -251,20 +266,35 @@ Below is a sequence diagram that illustrates the execution of `viewMeeting` comm
 
 #### 3.2.4 List Meeting feature
 
-Syntax: `listMeeting`
+Syntax: `listMeeting [d/PERIOD]`
 
-Purpose: View all `Meeting` from the Meeting List in `Model`.
+Purpose: View all `Meeting` from the Meeting List in `Model`, optionally using `d/PERIOD` to filter out `Meeting` from a specific time period.
 
 ##### Implementation
 
 Usage Scenario of `listMeeting`:
 
-1) User inputs `listMeeting` to view the current meetings in the `Model`.
+Step 1. User inputs `listMeeting d/tomorrow` to view the current meetings in the `Model`.
```

Comment:
just a suggestion, maybe `to view meetings that happen on tomorrow` will be a clearer description?

Reply:
Thanks for spotting that, it's supposed to be updated to reflect the tomorrow